spec/factories/orders.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :order do
+    product_name { Faker::Lorem.word }
+    product_count { Faker::Number.number(digits: 3).to_i }
```

Comment:
the Faker gem is a lot of fun, but you can write making a random integer using just Ruby too!